packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -330,23 +519,23 @@ export default class StatsAnalyzer extends EventsScope {
 
     const mediaType = type;
 
-    if (!this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver) {
-      this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver = result.packetsLost;
-    }
+    if (!this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver) { this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver = result.packetsLost; }
 
     const currentPacketLoss = result.packetsLost - this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver;
 
     this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver = result.packetsLost;
     this.statsResults[mediaType].send.packetsLostOnReceiver = currentPacketLoss;
     this.statsResults[mediaType].send.totalPacketsLostOnReceiver = result.packetsLost;
-    this.statsResults[mediaType].send.jitter = result.jitter;
+    this.statsResults[mediaType].send.remoteJitter = result.jitter;
     this.statsResults[mediaType].send.roundTripTime = result.roundTripTime;
-    this.packetLossArraySender[mediaType].push(currentPacketLoss);
+    this.statsResults[mediaType].send.timestamp = result.timestamp;
+    this.statsResults[mediaType].send.ssrc = result.ssrc;
+    this.statsResults[mediaType].send.reportsReceived = result.reportsReceived;
 
     // Total packloss ratio on this video section of the call
-    this.statsResults[mediaType].send.overAllPacketLossRatio = this.statsResults[mediaType].send.totalPacketsLostOnReceiver > 0 ? this.statsResults[mediaType].send.totalPacketsLostOnReceiver * 100 / this.statsResults[mediaType].send.totalPacketsSent : 0;
+    this.statsResults[mediaType].send.overAllPacketLossRatio = this.statsResults[mediaType].send.totalPacketsLostOnReceiver > 0 ? this.statsResults[mediaType].send.totalPacketsLostOnReceiver / this.statsResults[mediaType].send.totalPacketsSent : 0;
```

Comment:
Oh no, does this mean we are over downgrading currently?

Reply:
The downgrade logic uses different logic so i guess its should be fine